db/scripts/init_tables.sql

```diff
@@ -81,11 +81,12 @@ CREATE TABLE IF NOT EXISTS domain_totals
     time            TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
     source          TEXT NOT NULL,
     attempted       INTEGER DEFAULT 0,
-    connected       INTEGER DEFAULT 0,
     mta_sts_testing INTEGER DEFAULT 0,
```

Comment:
```suggestion
    with_mxs         INTEGER DEFAULT 0,
    mta_sts_testing INTEGER DEFAULT 0,
```
add the column here too, for reference